src/js/common.ml

```diff
@@ -109,3 +123,10 @@ let set_actor_aliases entries =
   let aliases = Flags.actor_aliases in
   aliases := Flags.M.of_seq (Array.to_seq entries)  
 
+let gc_flags option =
+  match Js.to_string option with
+  | "force" -> Flags.force_gc := true
+  | "scheduling" -> Flags.force_gc := false
+  | "copying" -> Flags.gc_strategy := Mo_config.Flags.Copying
+  | "marking" -> Flags.gc_strategy := Mo_config.Flags.MarkCompact
```

Comment:
Would it make sense to use the same names as moc compiler options, just for sanity?

Reply:
I think this is close enough. There is no sanity in JS...I just spent the last two days figuring out that `await setX(42)` in React doesn't guarantee x is 42 after the await...